core/server/api/v2/utils/validators/input/schemas/tags-add.json

```diff
@@ -0,0 +1,23 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "tags.add",
+    "title": "tags.add",
+    "description": "Schema for tags.add",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "tags": {
+            "type": "array",
+            "minItems": 1,
```

Comment:
Not sure about restricting the input array to `1` here. It makes sense, but might be too restrictive?

Reply:
makes sense for now